<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="confirmBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>確定</string>
    </property>
   </widget>
   <widget class="QLabel" name="chLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="funcWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>181</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="windowWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
